stable/instana-agent/README.md

```diff
@@ -122,6 +122,9 @@ The following table lists the configurable parameters of the Instana chart and t
 | `agent.leaderElectorPort`          | Instana leader elector sidecar port                                     | `42655`                                                                                                     |
 | `agent.listenAddress`              | List of addresses to listen on, or "*" for all interfaces               | `nil`                                                                                                       |
 | `agent.mode`                       | Agent mode (Supported values are APM, INFRASTRUCTURE, AWS)              | `APM`                                                                                                       |
+| `agent.updates.version`            | Which version of the updates pack to use without further updates (version pinning), specified as the hash of one of the commits of the [Instana Agent Updates](https://github.com/instana/agent-updates/commits/public) repository. This setting is applicable only to the [`instana/agent` image](https://hub.docker.com/r/instana/agent/) and its derivatives. If specified, this setting overrides `agent.updates.dynamic.frequency` and `agent.updates.dynamic.time`.                              | `{}`                                                                                                        |
```

Comment:
Should this be "...overrides 'agent.updates.frequency' and 'agent.updates.time'"? I dont see an `agent.updates.dynamic`. Same with the comment in the values.yaml below.

Reply:
yup